### PR TITLE
Tone down the watcher error

### PR DESCRIPTION
**Pull request by paulcsmith**
It took way too much attention away from the real error.

Make it subtle and less wordy.

Screen Shot 2019-11-15 at 5 56 54 PM

### CODE REVIEW

File: tasks/watch.cr [hunk 1]
```diff
@@ -166,15 +166,16 @@ module LuckySentry
     end
 
     private def print_error_message
-      puts "There was a problem compiling. Watching for fixes...".colorize(:red)
```
Comment (paulcsmith) [inline tasks/watch.cr]: Yeah I know. There's a giant compiler error 🤣

File: tasks/watch.cr [hunk 2]
```diff
@@ -166,15 +166,16 @@ module LuckySentry
...
       if successful_compilations.zero?
         puts <<-ERROR
 
-        Try this...
+        #{"---".colorize.dim}
```
Comment (paulcsmith) [inline tasks/watch.cr]: Divider so it is separate from the Crystal compile error

File: tasks/watch.cr [hunk 3]
```diff
@@ -166,15 +166,16 @@ module LuckySentry
...
 
-          #{green_arrow} If you haven't done it already, run #{"script/setup".colorize(:green)}
-          #{green_arrow} Run #{"shards install".colorize(:green)} to ensure dependencies are installed
-          #{green_arrow} Ask for help in #{"https://gitter.im/luckyframework/Lobby".colorize(:green)}
+        Feeling stuck? Try this...
+
+          ▸  Run setup: #{"script/setup".colorize.bold}
+          ▸  Reinstall shards: #{"rm -rf lib bin && shards install".colorize.bold}
```
Comment (paulcsmith) [inline tasks/watch.cr]: Sometimes cached binaries and shards cause issues during Crystal upgrades

File: tasks/watch.cr (hunk 2 above)
Comment (jwoertink) [reply tasks/watch.cr]: ooo nice

**Comment by paulcsmith** [comment]
Here's what it used to look like

Screen Shot 2019-11-15 at 6 03 27 PM

**Pull request merged**